### PR TITLE
update the referrals URLs to match the implementation in the API

### PR DESCRIPTION
**Pull request by tomsmyers**
## What does this pull request do?

- changes the URLs in the interventions service to match the URL structure discussed here https://mojdt.slack.com/archives/C01DYKJUKDX/p1606925013029800
- makes the id parameter more representative of the real IDs (in fact, passing plain integers to these endpoints will result in 400 responses - is this something we want/need to test in the UI layer?)

## What is the intent behind these changes?

ensure the api calls succeed on the real interventions UI when it's deployed.

### CODE REVIEW

File: server/services/interventionsService.test.ts [hunk 1]
```diff
@@ -16,26 +16,27 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     beforeEach(async () => {
       // This is just an example for getting Pact set up; we’ll properly design this endpoint later
       await provider.addInteraction({
-        state: 'There is an existing referral with ID of 1',
+        state: 'There is an existing referral with known id',
```
Comment (lawrence-forooghian) [inline server/services/interventionsService.test.ts:19]:
```suggestion
        state: 'There is an existing referral with ID of ac386c25-52c8-41fa-9213-fcf42e24b0b5`,
```
won't it be useful for you to know the exact ID that needs to be available when you need to set up this state in the backend Pact tests?

File: server/services/interventionsService.test.ts [hunk 2]
```diff
@@ -16,26 +16,27 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
...
-          path: '/referrals/1',
+          path: '/draft-referral/ac386c25-52c8-41fa-9213-fcf42e24b0b5',
           headers: { Accept: 'application/json' },
         },
         willRespondWith: {
           status: 200,
           body: Matchers.like({
-            id: '1',
+            id: 'ac386c25-52c8-41fa-9213-fcf42e24b0b5',
+            created: '2020-12-02 17:20:14',
```
Comment (lawrence-forooghian) [inline server/services/interventionsService.test.ts:30]: The addition of `created` doesn't seem related to the rest of the changes here, should it be here? Also what format is this datetime? It's not a valid ISO 8601 value.

**Review by lawrence-forooghian — CHANGES_REQUESTED** [review]
As mentioned in our commits RFC please could you make sure that the information which is in the pull request message (especially the motivaition) is also in the commit messages?

File: server/services/interventionsService.test.ts (hunk 2 above)
Comment (tomsmyers) [reply server/services/interventionsService.test.ts:30]: will remove and add in a seperate PR. good point on ISO formatting. it's the default serialization of java `Date`. i sort of figured it would be ISO formatting and did not notice it wasn't.

**Comment by lawrence-forooghian** [comment]
@tomsmyers is this ready for re-review?

**Review by lawrence-forooghian — APPROVED** [review]
